src/bin/index.js

```diff
@@ -26,21 +26,46 @@ const argv = yargs
     },
   })
   .options({
-    deep: {
+    'sort-definitions': {
```

Comment:
No need to repeat `options`. Multiple options can be defined within a single `options` invocation.